crates/sequencer/src/rpc.rs

```diff
@@ -31,6 +32,7 @@ pub(crate) struct RpcContext<C: sov_modules_api::Context> {
 
 pub(crate) fn create_rpc_module<C: sov_modules_api::Context>(
     rpc_context: RpcContext<C>,
+    da_db_path: Option<PathBuf>,
```

Comment:
I think we can remove da_publishBlock method from the sequencer and not pass this here. we don't really test locally on mock da anymore.

if any tests fail due to this find where in the test da_publishBlock rpc was called and replace that with da_service.publish_block (or whatever it's called)